src/constants/protocols.js

```diff
@@ -1,4 +1,4 @@
 export const ANALOG = 'analog';
 export const MTX = 'mtx';
```

Comment:
```suggestion
export const MTX1 = 'mtx1';
```

should we switch to this?

Reply:
use names same as in jooby-codec